app/controllers/sections_controller.rb

```diff
@@ -8,23 +8,42 @@ def index
     redirect_to courses_path
   end
 
+  def new
+    @course = Course.find(params[:course_id])
+  end
+
+  def create
+    @course = Course.find(params[:course_id])
+    @section = Section.new(section_params)
+    if @section.save
+      redirect_to course_path(@course), notice: 'Section saved.'
```

Comment:
Yup sounds good. I ended up having to throw ivars in because errors aren't displayed if the save fails (need the ivar to pass over to the view during render). Didn't realize that they even mattered for `create` and `update` actions. Learn something new every day!